Library/Formula/safe-rm.rb

```diff
@@ -1,12 +1,16 @@
-require 'formula'
+require "formula"
 
 class SafeRm < Formula
-  homepage 'https://launchpad.net/safe-rm'
-  url 'https://launchpad.net/safe-rm/trunk/0.10/+download/safe-rm-0.10.tar.gz'
-  sha1 '6b829ae68e1fa3c8016e15ab37fcc08caef7712f'
+  homepage "https://launchpad.net/safe-rm"
+  url "https://launchpad.net/safe-rm/trunk/0.10/+download/safe-rm-0.10.tar.gz"
+  sha1 "6b829ae68e1fa3c8016e15ab37fcc08caef7712f"
+
+  head do
+    url "https://gitorious.org/safe-rm/mainline.git", :branch => 'master'
```

Comment:
The ` branch ` is redundant here I think. ` Master ` is the default AFAIK. Thanks for the double quotes by the way! Appreciate that :+1: